samples/java_springboot/52.teams-messaging-extensions-search-auth-config/src/main/java/com/microsoft/bot/sample/teamssearchauth/TeamsMessagingExtensionsSearchAuthConfigBot.java

```diff
@@ -16,6 +16,7 @@
 import com.microsoft.bot.schema.*;
```

Comment:
Check these imports, should be individuals imports

Reply:
Done